src/repositories/cdn.js

```diff
@@ -53,7 +52,7 @@ class Cdn {
   }
 
   _getDefaultMetadata() {
-    return { createdon: moment.utc().toISOString() };
+    return { createdon: new Date().toISOString() };
```

Comment:
`new Date()` in JS return a local date, not UTC!

Reply:
Are we fine now like this or?

Reply:
Yes